src/htss/valentina/calcuator.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from unittest.mock import patch
+
+class NegativeValueError(Exception):
+    pass
+class OddValueError(Exception):
+    pass
+
+def calculate():
+    try:
+        num1 = int(input("Input the first number: "))
+        num2 = int(input("Input the second number: "))
+        total = num1 + num2
```

Comment:
If calculator errors on negative or odd numbers, we should do our checks on the inputs rather than on the total